docs/components/icons/index.md

```diff
@@ -10,10 +10,6 @@ Warp's icon set is designed to help users understand actions, information and dr
 
 <components-status react='released' vue='released' elements='released' />
 
-## Usage
-
-<component-design-guidelines name="Warp - Components / Icons" link="https://www.figma.com/file/yEx16ew6S0Xgd579dN4hsM/Warp---Icons?type=design&node-id=6011-1442&mode=design&t=zY5N398IPei2z89J-0" />
```

Comment:
I thought this Figma project is supposed to reflect what our @warp-ds/icons package supports, and if something is a work in progress (new icons are added or old icons are updated), the designers are supposed to use some branches in Figma, which are not visible to those who visit the page.

Reply:
Removed this based on the discussion we had the other day. Ref: https://sch-chat.slack.com/archives/C04P0GYTHPV/p1701078235668049?thread_ts=1701073116.821279&channel=C04P0GYTHPV&message_ts=1701078235.668049

Reply:
Later in that linked thread it was mentioned about the process including branches in Figma + that "After next icon update, what you see in figma and what you see in the tech docs will be in sync. But you can consider https://warp-ds.github.io/tech-docs/components/icons/ to be source of truth". Does it mean we should restore this link at some point ("next icon update")? @siljec